packages/matchbox/src/components/Popover/tests/Popover.test.js

```diff
@@ -1,175 +1,128 @@
 import React from 'react';
```

Comment:
Removed window event tests from this file – They are now a part of cypress tests, but are not reflected in coveralls / test coverage